content/influxdb/v2.0/reference/internals/file-system-layout.md

```diff
@@ -0,0 +1,250 @@
+---
+title: InfluxDB file system layout
+description: >
+  The InfluxDB file system layout depends on the operating system, package manager,
+  or containerization platform used to install InfluxDB.
+weight: 102
+menu:
+  influxdb_2_0_ref:
+    name: File system layout
+    parent: InfluxDB internals
+influxdb/v2.0/tags: [storage, internals]
+---
+
+The InfluxDB file system layout depends on the operating system, installation method,
+or containerization platform used to install InfluxDB.
+Anywhere InfluxDB is installed, its on-disk structure consists of the following:
+
+- [Engine path](#engine-path)
+- [Bolt path](#bolt-path)
+- [Configs path](#configs-path)
+- [InfluxDB configuration files](#influxdb-configuration-files)
+
+#### Engine path
+The **engine path** (referring to the [*storage engine*](/{{< latest "influxdb" >}}/reference/internals/storage-engine/)) is the directory path where InfluxDB stores time series data on disk.
+The engine path contains the following directories:
+
+- **data**: Time-Structured Merge Tree (TSM) files
+- **wal**: Write Ahead Log (WAL) files.
+
+Use the [engine-path](/influxdb/v2.0/reference/config-options/#engine-path)
+configuration option to customize the engine path.
+
+#### Bolt path
+The **bolt path** is the file path where InfluxDB stores the [Boltdb](https://github.com/boltdb/bolt) database.
+InfluxDB uses Boltdb as a file-based key-value store for non-time series data
+such as users, dashboards, tasks, etc.
+Use the [bolt-path](/influxdb/v2.0/reference/config-options/#bolt-path)
+configuration option to customize the bolt path.
+
+#### Configs path
+The **configs path** is the file path where InfluxDB stores
+[`influx` CLI connection configurations](/influxdb/v2.0/reference/cli/influx/config/) (configs).
+Use the `--configs-path` flag with `influx` CLI commands to set and use a custom
+configs path.
+
+#### InfluxDB configuration files
+Some operating systems and package managers store a default InfluxDB (`influxd`)
+configuration file on disk.
+For more information about using InfluxDB configuration files, see
+[Configuration options](/influxdb/v2.0/reference/config-options/).
+
+## File system layout per operating system
```

Comment:
@sanderson it may be easier to scan with two parallel headers, maybe InfluxDB file system and InfluxDB file system layout by OS? Taking the header up a level would streamline/tighten the view a bit too.